app/src/main/java/com/theopensourcefamily/chessclock/ClocksActivity.kt

```diff
@@ -35,7 +38,10 @@ class ClocksActivity : AppCompatActivity(), ClocksView {
   }
   override fun render() {
```

Comment:
I think the main idea of the satete is to be passed to this function as a parameter :) so the view should never hold a state val 💪
And to answer "when should the state be updated", the answer is everytime something has to be rendered, so for instance, everytime a clock ticks, the new state is pased to the view, who knows how to render it 🚀 
Does it make sense to you? we can give it a try :P